src/components/common/Toast/ToastProvider.jsx

```diff
@@ -0,0 +1,31 @@
+import { createContext, useContext, useState } from 'react';
+import Toast from './Toast';
+import styles from './Toast.module.scss';
+
+const ToastContext = createContext();
+
+export const useToast = () => useContext(ToastContext);
```

Comment:
💊 제안
해당 context를 사용하는 훅을 정의하셔서 사용성을 높으신 것 좋습니다.
다만 아래처럼 context 내부에서 해당 context에 접근하지 않을때의 에러도 추가해주시면 더 좋을 것 같습니다!

```suggestion
export const useToast = () => {
  const _context = useContext(ToastContext);

  if (!_context) throw new Error(/** error msg */);
  return _context;
}
```